includes/Modules/Analytics/Advanced_Tracking/Measurement_Events/FormidableForms_Event_List.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events\FormidableForms_Event_List
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Measurement_Events;
+
+/**
+ * Class for containing tracking event information for Formidable Forms plugin.
+ *
+ * @since n.e.x.t.
+ * @access ignore
+ * @ignore
+ */
+final class FormidableForms_Event_List extends Measurement_Event_List {
+
+	/**
+	 * FormidableForms_Event_List constructor.
+	 *
+	 * @since n.e.x.t.
+	 */
+	public function __construct() {
+		$event = new Measurement_Event(
+			array(
+				'pluginName' => 'Formidable Forms',
+				'category'   => 'engagement',
+				'action'     => 'form_submit',
+				'selector'   => '.frm_fields_container .frm_button_submit',
+				'on'         => 'click',
+				'metadata'   => <<<CALLBACK
+function( params, element ) {
+	var formId = element.closest('.frm_fields_container').querySelector('input[name="form_id"]').value;
```

Comment:
`closest` may not be available in older browser (particularly IE). Typically in Site Kit we don't support older browsers, but since this code will run on the site's frontend for _any_ visitors, we'll need to.

Let's implement `closest` function in the original snippet in `Measurement_Code_Injector`, similar to: https://github.com/ampproject/amphtml/blob/1d29dfbc5ebb1bf89a77091afaa2a712457c1db6/src/dom.js#L297